apidoc/Titanium/UI/iOS/Stepper.yml

```diff
@@ -0,0 +1,180 @@
+             singletap, swipe, touchcancel, touchend, touchmove, touchstart, twofingertap]
+properties:
+  - name: value
+    summary: The current value of the stepper.
+    type: Number
+
+  - name: continous
+    summary: |
+        If YES, value change events are sent immediately when the value changes during user interaction.
+        If NO, a value change event is sent when user interaction ends.
+        The default value is YES.
+    type: Boolean
+
+  - name: autorepeat
+    summary: |
+        If YES, the user pressing and holding on the stepper repeatedly alters value.
+        The default value is YES.
+    type: Boolean
+
+  - name: wraps
+    summary: |
+        If YES, incrementing beyond maximumValue sets value to minimumValue; likewise, decrementing below
+        minimumValue sets value to maximumValue. If NO, the stepper does not increment beyond maximumValue nor
+        does it decrement below minimumValue but rather holds at those values.
+        The default value is NO.
+    type: Boolean
+
+  - name: minimumValue
+    summary: The miniumum value the stepper will be set to, the value must be smaller than the maximumValue.
+    type: Number
+
+  - name: minimumValue
+    summary: The maximum value the stepper will be set to, the value must be greater than the miniumumValue.
+    type: Number
+
+  - name: minimumValue
+    summary: The value the stepper will decrement and increment by.
+    type: Number
+
+  - name: tintColor
+    summary: Sets the color for the widget, any backgroundImages added will be set to the same color.
+    description: |
+        Default color is blue
+        For information about color values, see the "Colors" section of <Titanium.UI>.
+    type: String
+
+  - name: backgroundImage
+    summary: |
+        Background image for the stepper in its normal state, specified as a local file path or URL.
+    description: |
+        Also sets the background image for other stepper states (focused, selected),
+        unless they've been specified explicitly. On iOS, if backgroundDisabledImage is unset, the
+        image will appear faded to indicate being disabled. Also on iOS, if backgroundSelectedImage
+        is unset, the image will be darkened to indicate being selected.
+    type: String
+
+  - name: backgroundDisabledImage
+    summary: |
+        Background image for the stepper in its disabled state, specified as a local file path
+        or URL. 
+    description: |
+        Also sets the background image for the other stepper states (normal, focused, selected), 
+        unless they've been specified explicitly, except for iOS, which requires a backgroundImage.
+    type: String
+    platforms: [android, iphone, ipad, mobileweb]
+
+  - name: backgroundSelectedImage
+    summary: |
+        Background image for the stepper in its selected state, specified as a local file 
+        path or URL.
+    type: String
+    platforms: [iphone, ipad]
```

Comment:
Similar as above: As the API is iOS-only, we don't need to explicitly declare the platform support